homeassistant/components/ping/device_tracker.py

```diff
@@ -125,7 +125,7 @@ async def async_update(now):
                     count=PING_ATTEMPTS_COUNT,
                     timeout=ICMP_TIMEOUT,
                     privileged=privileged,
-                    id=async_get_next_ping_id(hass),
+                    id=async_get_next_ping_id(hass, len(ip_to_dev_id.keys())),
```

Comment:
No need to pass keys.

```suggestion
                    id=async_get_next_ping_id(hass, len(ip_to_dev_id)),
```